skglm/tests/test_penalties.py

```diff
@@ -4,12 +4,15 @@
 from numpy.linalg import norm
 from numpy.testing import assert_array_less
 
+from slope.solvers import pgd_slope
```

Comment:
Is there a reason for not importing these locally in ``test_slope`` and using ``.xfail`` if the dependence is not installed?

Cf. 
https://github.com/scikit-learn-contrib/skglm/blob/92c2a456b360a923f4cb961cea2a97c417fc13c2/skglm/experimental/tests/test_sqrt_lasso.py#L19-L23

Reply:
no. i'll change it, thx :)